extensions/oidc/runtime/src/main/java/io/quarkus/oidc/TokenIntrospection.java

```diff
@@ -21,6 +25,43 @@ public TokenIntrospection(JsonObject json) {
         super(json);
     }
 
+    public boolean isActive() {
+        return getBoolean(OidcConstants.INTROSPECTION_TOKEN_ACTIVE);
+    }
+
+    public String getUsername() {
+        return getString(OidcConstants.INTROSPECTION_TOKEN_USERNAME);
+    }
+
+    public String getSubject() {
+        return getString(OidcConstants.INTROSPECTION_TOKEN_SUB);
+    }
+
+    public String getAudience() {
+        return getString(OidcConstants.INTROSPECTION_TOKEN_AUD);
+    }
+
+    public String getIssuer() {
+        return getString(OidcConstants.INTROSPECTION_TOKEN_ISS);
+    }
+
+    public Set<String> getScopes() {
+        if (this.contains(OidcConstants.TOKEN_SCOPE)) {
+            Set<String> scopes = new HashSet<>();
```

Comment:
We know the size here, so best to set it

Reply:
Hi @geoand Sure, thanks for keeping an eye on these things, have a look please if I did it right now :-)